rust-runtime/aws-smithy-http/src/event_stream/receiver.rs

```diff
@@ -153,6 +152,8 @@ impl<T, E> Receiver<T, E> {
     }
 
     async fn buffer_next_chunk(&mut self) -> Result<(), SdkError<E, RawMessage>> {
+        use http_body::Body;
```

Comment:
could make this `http_body_02x` to make it consistent where you've done that elsewhere